examples/formbot/domain.yml

```diff
@@ -77,35 +78,39 @@ responses:
   utter_restart:
     - text: "restarted"
 
+actions:
+- action_submit_restaurant_form
+- validate_restaurant_form
+
 forms:
   restaurant_form:
-      cuisine:
-        - type: from_entity
-          entity: cuisine
-          not_intent: chitchat
-      num_people:
-        - type: from_entity
-          entity: number
-          intent: [inform, request_restaurant]
-      outdoor_seating:
-        - type: from_entity
-          entity: seating
-        - type: from_intent
-          intent: affirm
-          value: true
-        - type: from_intent
-          intent: deny
-          value: false
-      preferences:
-        - type: from_intent
-          intent: deny
-          value: no additional preferences
-        - type: from_text
-          not_intent: affirm
-      feedback:
-        - type: from_entity
-          entity: feedback
-        - type: from_text
+    cuisine:
+      - type: from_entity
+        entity: cuisine
+        not_intent: chitchat
+    num_people:
+      - type: from_entity
+        entity: number
+        intent: [inform, request_restaurant]
```

Comment:
Is this the proper format?
I thought lists in yaml are done as:

```
intent:
 - inform
 - request_restaurant

Reply:
It's technically valid, but yeah this doesn't follow best conventions and needs to be updated.